src/hero/_base.scss

```diff
@@ -1,22 +1,30 @@
-.banner {
-  padding: 50px 23px;
+/**
+ * @file Hero
+ * @module Hero
+ * @overview Hero and top of page featured hero modules
+ */
 
-  .user-info {
-    font-weight: 400;
-    font-size: 20px;
-  }
+.hero {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  min-height: 300px;
+  padding: ms(8) ms(2);
+  text-align: center;
+  background-size: cover;
+  background-position: center center;
+}
 
-  @include respond-at-and-above($break-large) {
-    padding-top: 78px;
-    padding-bottom: 78px;
-  }
+.hero--featured {
+  justify-content: flex-start;
+  height: 90vh;
 }
 
-.banner-title {
+.hero-heading {
   font-size: 30px;
   margin-bottom: 10px;
 
   @include respond-at-and-above($break-medium) {
     font-size: 36px;
   }
-}
\ No newline at end of file
+}
```

Comment:
@josephco For future, let's always resort to our ms values whenever we can especially during prototyping.